src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      calculator.result += num;
+
+      return this;
+    },
+    subtract(num) {
+      calculator.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      calculator.result *= num;
+
+      return this;
+    },
+    divide(num) {
+      calculator.result /= num;
+
+      return this;
+    },
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+    operate(callback, number) {
+      callback(number);
```

Comment:
Use 'this' as the context for the callback function by calling 'callback.call(this, number)'